xain_fl/coordinator/store.py

```diff
@@ -42,6 +42,30 @@ def read_weights(self, participant_id: str, round: int) -> ndarray:
         """
 
 
+class DummyStore(AbstractStore):
```

Comment:
perhaps more suggestive naming would be better, to distinguish `FakeS3Store` and `DummyStore` better. e.g. something like `LocalS3Store` and `MockStore`?

Reply:
To me, "mock" sounds like something we use in tests, so I'd prefer not to rename `DummyStore` into `MockStore`. Following that logic, I did rename `FakeS3Store` into `MockS3Store` though. Is that better? @Robert-Steiner do you have an opinion regarding the names here?

Reply:
Instead of `Dummy` it is sometimes called `NullObject` but for me it is fine to call it `DummyStore` and `MockStore`.

Reply:
I think `NullStore` (or similar) would be more descriptive

Reply:
Let's go for `NullObjectStore`. I'll also make the change in `xain-sdk` for consistency.

Reply:
Done